### ISSUE TITLE
Can not verify if and which with connected.

### ISSUE THREAD

**Issue by Erf-** [rockymeza/wifi#79]
Hi Rocky. It seems the app does not have any command or class method to return connection state. It would be nice to have it done with the eventual network ssid we are connected with.
Do you want me to open a pull requestfor that? I have some ideas I could apply.
PS: I have just noticed scheme.activate() issue has not been fixed yet.

**Comment by rockymeza** [comment]
Hey @Erf-,

Definitely want to have this functionality.

I have started some functionality like that in #44, if you want some inspiration. I'm not a huge fan of using iwgetid as I don't know which distros it's available on and I'm not super happy about adding more external dependencies.

**Comment by Erf-** [comment]
You are right. I suggest having the current scheme written in a config file. Then we could extend the config file to any running config data. I am looking for an existing file you would have already created that is used as I described. Do you have any suggestion about it ?
